Hero.py

```diff
@@ -249,6 +249,19 @@ def heroperks(self):
             self.levelupaug = .6
             # 50% crit chance boost
             self.critaug = 50
+        elif self.ourclass == 'barbarian':  # defined by high base damage and low defense
+            # decent health
+            self.hpaug = 11
+            # low dodge
+            self.dodgeaug = 4
+            # low DEF
+            self.defaug = 6
+            # high atk
+            self.atkaug = 1
```

Comment:
This needs to be higher if he has a high attack aug. The mage has an attack aug of 12 for reference as a glass cannon.